### DISCUSSION TITLE
Cant export SKU in Pimcore

### DISCUSSION THREAD

**Discussion by pmcr234** [pimcore/pimcore#17651] [Q&A / Support]
Hey I need to export my order data. I want to include my product sku however pimcore which I use with coreshop doesnt allow me to do that. Please help, it's an urgent matter!

<img width="809" alt="Bildschirmfoto 2024-09-16 um 17 30 52" src="https://github.com/user-attachments/assets/8ba36476-58dd-4425-957c-9a669f51213a">
<img width="815" alt="Bildschirmfoto 2024-09-16 um 17 30 15" src="https://github.com/user-attachments/assets/452b5a99-c4e5-469a-8e21-03bb7dc26cba">
<img width="593" alt="order 2" src="https://github.com/user-attachments/assets/a8d012ae-409d-4d01-8c4c-c14eed2bb722">
<img width="597" alt="order item 1" src="https://github.com/user-attachments/assets/3a989ca6-5d54-4248-8b8a-46e1a60ad6f9">
<img width="605" alt="coreshop product 3" src="https://github.com/use
<img width="821" alt="coreshop product" src="https://github.com/user-attachments/assets/4a2c2aa8-4764-4574-b4d6-6b8c24e6d254">
r-attachments/assets/bf2e433e-081f-43bb-8e7c-0b8c6dd830fa">
<img width="598" alt="coreshop product 2" src="https://github.com/user-attachments/assets/e35d8619-7b07-4394-8837-828cb21042a7">
<img width="605" alt="coreshop product 1" src="https://github.com/user-attachments/assets/17bc39ed-aa47-42f9-8f8a-47c54434ce89">

**Comment by dpfaffenbauer** [comment]
Hey @pmcr234  opening another thread doesn't resolve your issue any faster (https://github.com/orgs/pimcore/discussions/17619).

If you need urgent support, you can get paid support from us at CoreShop. Please contact us directly here www.coreshop.org.

**Comment by kingjia90** [comment]
Closing as duplicate of https://github.com/orgs/pimcore/discussions/17619

**Reply by pmcr234** [reply]
I haven't gotten a reply in almost a week, wherefore I opened another thread, maybe the participant isn't there anymore but I still need support.

**Reply by kingjia90** [reply]
Understandable but you could just write a gentle reminder under the other thread, and the thread would be pinned up again as it's sort by last interaction (by default by GitHub) gaining the potential attention of new participants without having everyone to reconstruct the history in multiple threads and "losing" important info on the way eg. what was already checked or like that the issue is encountered in pimcore `6.6.1`.
